actions/team_actions.jsx

```diff
@@ -91,14 +91,18 @@ export function addUserToTeamFromInvite(data, hash, inviteId, success, error) {
     );
 }
 
-export async function addUsersToTeam(teamId, userIds, success, error) {
-    const {data: teamMembers, error: err} = await TeamActions.addUsersToTeam(teamId, userIds)(dispatch, getState);
-    getChannelStats(ChannelStore.getCurrentId())(dispatch, getState);
-    if (teamMembers && success) {
-        success(teamMembers);
-    } else if (err && error) {
-        error({id: err.server_error_id, ...err});
-    }
+export function addUsersToTeam(teamId, userIds) {
+    return async (doDispatch, doGetState) => {
+        const {data, error} = await doDispatch(TeamActions.addUsersToTeam(teamId, userIds));
+
+        if (error) {
+            return {error};
+        }
+
+        doDispatch(getChannelStats(doGetState().entities.channels.currentChannelId));
```

Comment:
Is there a socket listener for `user added to team`?  I see a listener here which would call `ChannelStats` but there is not corresponding emit from the server when I added a user. 

https://github.com/mattermost/mattermost-webapp/blob/ad3366d09c503ed2e46bb1dd16568f84f5eb54ea/actions/websocket_actions.jsx#L454

 This causes a really minor sideeffect of tab sync in channel members but highly unlikely given both tabs should have same default channel in view. 

If there exists a socket event then we can remove this and move it to the respective event.